packages/evalite/src/types.ts

```diff
@@ -743,4 +750,118 @@ export declare namespace Evalite {
       suites: Suite[];
     };
   }
+
+  /**
+   * Types for scorers and scorer-related functionality.
+   */
+  export namespace Scorers {
+    export interface SingleTurnData {
```

Comment:
I think this single-turn data type and multi-turn data type doesn't make sense. I think instead, each scorer should simply declare what it needs on the scorer itself and have a type dedicated to that scorer.